src/object/obj_rpc.c

```diff
@@ -32,7 +32,7 @@
 #include "obj_rpc.h"
 #include "rpc_csum.h"
 
-static int
+static inline int
```

Comment:
Do not add inline, the compiler will be better at working out the requirements than anyone else.

Reply:
This is additional hint to it. :-)